src/clj/game/cards/programs.clj

```diff
@@ -588,6 +588,11 @@
                           :effect (effect (update! (assoc card :hosted-programs (remove #(= (:cid target) %) (:hosted-programs card))))
                                           (lose :memory (:memoryunits target)))}}}
 
+   "Reaver"
+   {:events {:runner-trash {:req (req (and (first-event state :runner :runner-trash) (:installed target)))
```

Comment:
If in fact it turns out this is meant to work on Corp cards, use `(installed? target)`, because only Runner cards ever have `:installed true`.